doc/nrf/libraries/others/flash_map_pm.rst

```diff
@@ -0,0 +1,55 @@
+.. _lib_flash_map_pm:
+
+Partition Manager flash map
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Partition Manager flash map library provides a set of defines for reading flash partition information when using the :ref:`partition_manager` script.
+This is mandatory for providing a unified interface for flash partitions when Partition Manager is used.
+
+Overview
+********
+
+In Zephyr, :file:`flash_map.h` provides unified interface to code for reading the flash area data, such as offset, size, and device.
```

Comment:
Is "provides unified interface" correct? Seems odd to me, but I'm no expert.

Reply:
I agree this is odd. @de-nordic, how would you summarize this API's purpose?